database_schemas/cassandra/benchflow.cql

```diff
@@ -87,7 +87,7 @@ CREATE TABLE process (
   to_ignore boolean,
   experiment_id text,
   trial_id text,
-  PRIMARY KEY ((experiment_id, trial_id), source_process_instance_id, process_instance_id)
+  PRIMARY KEY ((experiment_id, trial_id), process_instance_id)
```

Comment:
How is it possible? This is the id the source database uses to differentiate entities